packages/grid/src/internal/TopRightPart.css

```diff
@@ -0,0 +1,17 @@
+.uitkGridTopRightPart {
+  position: absolute;
+  top: 0;
+  right: var(--uitkGrid-scrollBarWidth);
+  z-index: 2;
+}
+
+.uitkGridTopRightPart-raised {
+  box-shadow: -6px 0 4px -4px var(--uitkGrid-shadow-color);
+}
+
+.uitkGridTopRightPart table {
+  table-layout: fixed;
+
+  width: var(--uitkGrid-rightWidth);
```

Comment:
probably best to keep tokens like these as something like -rightPart-width?

Reply:
renamed